runtimevar/etcdvar/etcdvar.go

```diff
@@ -28,17 +28,29 @@ import (
 	"google.golang.org/grpc/codes"
 )
 
-const errWait = 10 * time.Second
+// defaultWait is the default value for WatchOptions.WaitTime.
+const defaultWait = 30 * time.Second
 
 // New constructs a runtimevar.Variable object that uses client to watch
 // variables in etcd.
 // Provide a decoder to unmarshal updated configurations into similar
 // objects during the Watch call.
-func New(name string, cli *clientv3.Client, decoder *runtimevar.Decoder) (*runtimevar.Variable, error) {
+func New(name string, cli *clientv3.Client, decoder *runtimevar.Decoder, opts *WatchOptions) (*runtimevar.Variable, error) {
+	if opts == nil {
+		opts = &WatchOptions{}
```

Comment:
nit:

```
if opts == nil {
  opts = &WatchOptions{ WaitTime: defaultWait }
}
```

And you can simplify the switch to be a single `if waitTime < 0`

Reply:
I copied this from the other implementations, so I'd rather leave it consistent.

I don't think we could remove the "waitTime == 0" from the switch, because someone could pass &WatchOptions{}, which is not nil but also has a 0 WaitTime that we need update the default for. For example, many of the conformance tests do this now.